client/ios/coolItWinter2019/Info.plist

```diff
@@ -55,6 +55,16 @@
 	</dict>
 	<key>UIAppFonts</key>
 	<array>
+<<<<<<< HEAD
```

Comment:
seems like you have merge conflicts and they are not resolved properly. Please do the rebasing properly and remove your merge conflicts.

Reply:
Resolved